Scoop/SearchRidesViewController.swift

```diff
@@ -0,0 +1,284 @@
+        }
+        
+        departureLabel.text = "Departure location"
+        departureLabel.snp.makeConstraints { make in
+            make.top.equalTo(departureTextField).inset(-labelTop)
+            make.leading.equalTo(departureTextField).inset(labelLeading)
+            make.height.equalTo(16)
+            make.width.equalTo(120)
+        }
+        
+        arrivalLabel.text = "Arrival location"
+        arrivalLabel.snp.makeConstraints { make in
+            make.top.equalTo(arrivalTextField).inset(-labelTop)
+            make.leading.equalTo(arrivalTextField).inset(labelLeading)
+            make.height.equalTo(16)
+            make.width.equalTo(99)
+        }
+        
+        departureDateLabel.text = "Departure date"
+        departureDateLabel.snp.makeConstraints { make in
+            make.top.equalTo(departureDateTextField).inset(-labelTop)
+            make.leading.equalTo(departureDateTextField).inset(labelLeading)
+            make.height.equalTo(16)
+            make.width.equalTo(99)
+        }
+    }
+    
+    @objc private func presentDepartureSearch() {
+        let depatureVC = DepartureSearchViewController()
+        depatureVC.delegate = self
+        navigationController?.pushViewController(depatureVC, animated: true)
+        departureLabel.isHidden = false
+    }
+
+    @objc private func presentArrivalSearch() {
+        let arrivalVC = ArrivalSearchViewController()
+        arrivalVC.delegate = self
+        navigationController?.pushViewController(arrivalVC, animated: true)
+        arrivalLabel.isHidden = false
+    }
+    
+    @objc private func updateDate() {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateStyle = .medium
+        departureDateTextField.text = dateFormatter.string(from: datePicker.date)
+        departureDateLabel.isHidden = false
+    }
+    
+    @objc private func presentMatches() {
+        guard let departure = self.departureTextField.textField.text, !departure.isEmpty,
+              let arrival = self.arrivalTextField.textField.text, !arrival.isEmpty,
+              let date = self.departureDateTextField.text, !date.isEmpty,
+              let departureID = departurePlace?.placeID,
+              let arrivalID = arrivalPlace?.placeID else {
+                  self.presentErrorAlert(title: "Error", message: "Please complete all fields.")
+                  return
+              }
+        
+        NetworkManager.shared.searchLocation(depatureDate: tripDate, startLocation: departureID, endLocation: arrivalID) { response in
+            switch response {
+            case .success(let response):
+                self.navigationController?.pushViewController(MatchesViewController(rides: response.rides), animated: true)
+            case .failure(let error):
+                print(error)
+            }
+        }
+    }
+
+    /** Converts a Date object into String format, where this one is specifically in the YYYY-MM-dd format. */
```

Comment:
Nice documentation! Personally I use `///` for a single-line doc and `/** */` for multi-line but that's just me.